src/amo/components/AddonReviewCard/index.js

```diff
@@ -58,7 +63,12 @@ type InternalProps = {|
   submittingReply: boolean,
 |};
 
-export class AddonReviewListItemBase extends React.Component<InternalProps> {
+export class AddonReviewCardBase extends React.Component<InternalProps> {
```

Comment:
Well, `AddonReview` is taken and it would be a harder patch to review if I renamed that one first and replaced it with this. Once we get rid of that component I'd be fine to name this one `AddonReview`. For the time being, I couldn't think of anything better than `AddonReviewCard`. Alternatives: `AddonReview2`, `AddonReviewNew`, `AddonReviewNoo`, `AddonReviewForReal` ... I could go on but I won't!